flycheck.el

```diff
@@ -6822,14 +6818,12 @@ See URL `https://golang.org/cmd/go'."
 (flycheck-define-checker go-test
   "A Go syntax and type checker using the `go test' command.
 
+Requires Go 1.6 or newer.
+
 See URL `http://golang.org/cmd/go'."
```

Comment:
Please fold both lines into a single paragraph.  Let's not use so much vertical space in docstrings, as the help buffers of syntax checkers are pretty stuffed with text already. :blush: